pkgs/tools/backup/tarsnapper/default.nix

```diff
@@ -35,23 +39,15 @@ python3Packages.buildPythonApplication rec {
     pexpect
   ];
 
-  doCheck = python3Packages.pythonOlder "3.12";
-
   nativeCheckInputs = with python3Packages; [
-    nose
+    pytestCheckHook
   ];
 
   # Remove standard module argparse from requirements
   pythonRemoveDeps = [ "argparse" ];
 
   makeWrapperArgs = [ "--prefix PATH : ${lib.makeBinPath [ tarsnap ]}" ];
 
-  checkPhase = ''
-    runHook preCheck
-    nosetests tests
-    runHook postCheck
-  '';
-
```

Comment:
This is unrelated, could you do this in a separate commit? That's generally good practice in case we need to revert anything later.

Reply:
This is related to dropping `nose` tests